packages/experiments/src/components/Button/examples/Button.Example.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Button } from '../index';
-import { HorizontalStack, Text, VerticalStack } from '@uifabric/experiments';
+import { HorizontalStack, VerticalStack } from 'office-ui-fabric-react/lib/Stack';
```

Comment:
Also can be moved into line 5. Basically consolidate stack imports into any existing import from OUFR lines.